### DISCUSSION TITLE
zone integration other than sonos

### DISCUSSION THREAD

**Discussion by sebastiensoulac** [punxaphil/custom-sonos-card#318] [Q&A]
Bonjour @PunxsutawneyPhil

As agreed, I'm writing to you in GitHub.
As you can see, on the right I have your module and on the left my zone management from the C2N-AMP crestron.

![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/dd33c874-7dea-4372-9fff-6b3dae642117)

Maintenant je souhaite intégré dans votre module l'activation d'une zone et la gestion du volume issue du crestron C2N-AMP.
![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/07ae7f52-e8f3-434d-bedb-4c4ca7bd4d60)

Below is the code
`state_color: true
type: custom:collapsable-cards
title: Multiroom
cards:
  - type: entities
    entities:
      - entity: input_boolean.room1
        name: Room 1
        secondary_info: last-changed
      - entity: input_select.zone
      - entity: input_number.volume_room1
        type: custom:slider-entity-row
        step: 5
        min: 0
        max: 100
        grow: true
        full_row: true
      - entity: input_boolean.room2
        name: null
        secondary_info: last-changed
      - entity: input_number.volume_room2
        type: custom:slider-entity-row
        step: 5
        min: 0
        max: 100
        grow: true
        full_row: true
      - entity: input_boolean.room3
        name: null
        secondary_info: last-changed
      - entity: input_number.volume_room3
        type: custom:slider-entity-row
        step: 5
        min: 0
        max: 100
        grow: true
        full_row: true
      - entity: input_boolean.room4
        name: null
        secondary_info: last-changed
      - entity: input_number.volume_room4
        type: custom:slider-entity-row
        step: 5
        min: 0
        max: 100
        grow: true
        full_row: true
      - entity: input_boolean.room5
        name: null
        secondary_info: last-changed
      - entity: input_number.volume_room5
        type: custom:slider-entity-row
        step: 5
        min: 0
        max: 100
        grow: true
        full_row: true
      - entity: input_boolean.room6
        name: null
        secondary_info: last-changed
      - entity: input_number.volume_room6
        type: custom:slider-entity-row
        step: 5
        min: 0
        max: 100
        grow: true
        full_row: true`

Here's a synoptic diagram to help you understand.
![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/a4335e85-3d31-4496-8663-6dc87c6e900c)

I hope I've been as clear as possible

**Comment by punxaphil** [comment]
I'm afraid that won't work. This card only supports media_player.* entities.

**Reply by sebastiensoulac** [reply]
Thank you for your feedback.
If I convert to media_player.* entities could it be integrated?

**Reply by punxaphil** [reply]
Probably
